modules/core/src/transactionBuilder.ts

```diff
@@ -360,7 +379,11 @@ exports.createTransaction = function(params) {
 
   // Iterate unspents, sum the inputs, and save _inputs with the total
   // input amount and final list of inputs to use with the transaction.
-  let feeSingleKeyUnspentsUsed = [];
+  let feeSingleKeyUnspentsUsed: {
+    value: number;
+    tx_hash: Buffer;
+    tx_output_n: number;
+  }[] = [];
```

Comment:
this is repeated from lines 356-360. doesn't bother me too much, but might be worth defining the type somewhere up top instead

Reply:
done